library/Icingadb/Util/PerfData.php

```diff
@@ -449,15 +643,67 @@ protected function format($value)
         if ($this->isPercentage()) {
```

Comment:
* Consider to use a `switch (true) { case $this->isPercentage(): ... }` here
  * There are now so much similar control structures, it makes sense to condense them
* Also please put a space between casts and variables everywhere (i.e. `(string) $value`)